benchmark/common.js

```diff
@@ -280,7 +280,7 @@ function sendResult(data) {
     process.send(data);
   } else {
     // Otherwise report by stdout
-    console.log(formatResult(data));
+    process.stdout.write(formatResult(data));
```

Comment:
Should it have a comment to warn someone from changing it back at a future date?